_posts/2024-05-08-new-mailing-lists.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "New Mailing Lists for Sandstorm"
+author: Jacob Weisz
+authorUrl: https://github.com/ocdtrekkie
+---
+
+Several of us have long talked about how to get away from Google Groups and when to do
+so. Google has not gotten better at managing spam both in terms of junk registrations
```

Comment:
I'd put a comma after "spam". OTOH too many commas are run-on, so I'd maybe remove the one after "once,"?